apps/settings/elements/wifi-knowNetworks.html

```diff
@@ -0,0 +1,23 @@
+<element name="wifi-knowNetworks" extends="section">
```

Comment:
shouldn't the filename and name attribute on this line be knownNetworks, not knowNetworks?